src/index.coffee

```diff
@@ -18,7 +18,7 @@ class DatadogPlugin
     @ee.on 'done', @flushStats
 
   getDatadogConfig: ->
-    flushIntervalSeconds: 0
+    # flushIntervalSeconds: 0
```

Comment:
letting auto flush take over since that is how it was even with flushIntervalSeconds = 0 with v0.3.0 of datadog-metrics. datadog-metrics had a bug which was doing auto flush for flushIntervalSeconds = 0, but is now taken care of since 0.5.0.

Reply:
Identified a potential issue with "the last flush" - Artillery exits before it can be sent? For example, if the test is ~5 seconds, nothing gets sent to Datadog. This is true for both version `0.1.1` and the version in this PR. Needs further investigation.

Reply:
Good catch!